app/models/supported_ssg.rb

```diff
@@ -7,16 +7,25 @@
                           :os_major_version, :os_minor_version,
                           keyword_init: true) do
   self::SUPPORTED_FILE = Rails.root.join('config/supported_ssg.yaml')
+  OS_NAME = 'RHEL'
+  self::OS_NAME = OS_NAME
```

Comment:
Since this is not a proper class, this constant is created on a class and on an instance. Please suggest an alternative, if there is one.

Reply:
Yeah, I thought the same. We can refactor it post-hoc.